src/dsp_tools/commands/xmlupload/check_consistency_with_ontology.py

```diff
@@ -100,12 +100,15 @@ def _get_all_property_names_and_resource_ids_one_resource(
     resource: etree._Element, prop_dict: dict[str, list[str]]
 ) -> dict[str, list[str]]:
     for prop in resource.iterchildren():
-        if prop.tag != "bitstream":
-            prop_name = prop.attrib["name"]
-            if prop_name in prop_dict:
-                prop_dict[prop_name].append(resource.attrib["id"])
-            else:
-                prop_dict[prop_name] = [resource.attrib["id"]]
+        match prop.tag:
+            case "bitstream" | "iiif-uri":
+                pass
+            case _:
+                prop_name = prop.attrib["name"]
+                if prop_name in prop_dict:
+                    prop_dict[prop_name].append(resource.attrib["id"])
+                else:
+                    prop_dict[prop_name] = [resource.attrib["id"]]
```

Comment:
by passing an empty list as default to the `get()` method, this can be simplified:

```suggestion
                prop_name = prop.attrib["name"]
                previous = prop_dict.get(prop_name, [])
                prop_dict[prop_name] = previous.append(resource.attrib["id"])
```

Or you can inline this to be a 1 or 2 liner.  
It might even work (but I'm not sure) like this:

```python
                prop_name = prop.attrib["name"]
                prop_dict.get(prop_name, []).append(resource.attrib["id"])
```

Which then could definitely be turned into

```python
                prop_dict.get(prop.attrib["name"], []).append(resource.attrib["id"])
```